### PR TITLE
Add full yeelight remote control support (fancl5)

### PR DESCRIPTION
**Pull request by 0neday**
add full yeelight remote controller support now
repull request

### CODE REVIEW

File: yeelight_light_fancl5.yaml [hunk 1]
```diff
@@ -136,7 +226,64 @@ xiaomi_ylyk01yl:
...
+       - switch.turn_on: buzzer_beep
+       
+    # stepless cycle control speed
+    - keycode: 2
+      then:
+        - fan.cycle_speed: yeelight_ceiling_fan
+        - switch.turn_on: buzzer_beep
+       
+    # fan mode REVERSE select for circle winds
+    - keycode: 4
```
Comment (syssi) [inline yeelight_light_fancl5.yaml:242]: You are turning the fan on at long press (**keycode 4**)

File: yeelight_light_fancl5.yaml [hunk 2]
```diff
@@ -136,7 +226,64 @@ xiaomi_ylyk01yl:
...
+      then:
+        - fan.turn_on:
+           id: yeelight_ceiling_fan
+           direction: reverse
+        - switch.turn_on: buzzer_beep
+
+    # color_temperature control，only support by ceiling light (154K-370K)
+    # color_temperature minus
+    - keycode: 4
```
Comment (syssi) [inline yeelight_light_fancl5.yaml:251]: Here you change the color temperature also by **keycode 4**. This looks wrong to me.

**Comment by syssi** [comment]
Please fix the lint issues of the `yamllint` step (trailing spaces).

File: yeelight_light_fancl5.yaml (hunk 2 above)
Comment (0neday) [reply yeelight_light_fancl5.yaml:251]: wait me a second, I test again

File: yeelight_light_fancl5.yaml (hunk 1 above)
Comment (0neday) [reply yeelight_light_fancl5.yaml:242]: no, turn on fan using keycode 0

long press keycode 4 from change reverse mode, short press  keycode 4 for forward mode

**Comment by 0neday** [comment]
> Please fix the lint issues of the `yamllint` step (trailing spaces).

how fix, I using notepad++ and esphome editor

**Pull request closed without merging**